src/mobile/apps/artwork/components/meta_data/templates/price.jade

```diff
@@ -3,4 +3,5 @@ if artwork.sale_message
     if artwork.sale_message
       .sale_message= artwork.sale_message
     if artwork.price && artwork.availability !== 'sold'
-      .shipping-info Shipping, tax, and services quoted by seller
+      .shipping-info
+        | #{artwork.shippingInfo}
```

Comment:
ah yes, Martsy! 😅

Reply:
Also not that the logic here is slightly different. We do not account for on loan or permanent collection with price here but 🤷‍♀️